### PR TITLE
feat: add new util folder and func to generate rand number

### PR DESCRIPTION
**Pull request by jowjow22**
test

### CODE REVIEW

File: utils/generateRandomNumber.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+#include "libs/mtwister/definitions/mtwister.h"
+
+int randNum()
```
Comment (lucasmoraesb) [inline utils/generateRandomNumber.c:4]: its wrong jonas.

File: utils/generateRandomNumber.c [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+{
+  MTRand r = seedRand(1337);
+  int i;
+  for (i = 0; i < 1000; i++)
+  {
```
Comment (Elquiasjr) [inline utils/generateRandomNumber.c:9]: There's something in the way

File: utils/generateRandomNumber.c [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+#include "libs/mtwister/definitions/mtwister.h"
+
+int randNum()
+{
+  MTRand r = seedRand(1337);
+  int i;
```
Comment (lucasmoraesb) [inline utils/generateRandomNumber.c:7]: hi my name  is AAHHHH

**Review by lucasmoraesb — CHANGES_REQUESTED** [review]

**Review by Elquiasjr — CHANGES_REQUESTED** [review]
hello moto

File: utils/generateRandomNumber.c (hunk 3 above)
Comment (jowjow22) [reply utils/generateRandomNumber.c:7]: fixed at [7513ba6](https://github.com/jowjow22/simplified-c-blockchain/pull/1/commits/7513ba65563b425df211e982a9ce2f788d049b31)

File: utils/generateRandomNumber.c [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  for (i = 0; i < 1000; i++)
+  {
+    return genRand(&r);
+  }
+}
```
Comment (Elquiasjr) [inline utils/generateRandomNumber.c:12]: deixa os garoto brincar

**Review by Elquiasjr — CHANGES_REQUESTED** [review]

**Review by Elquiasjr — APPROVED** [review]

**Review by lucasmoraesb — APPROVED** [review]

**Pull request closed without merging**